### PR TITLE
Use Netlify redirects to create a simple URL to our installation files

### PR DESCRIPTION
**Pull request by scholzj**
Use Netlify redirects to create a simple URL to our installation files - somthing like `https://strimzi.io/install` to redirect to the single YAML installation.

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Redirects to the latest all-in-one install file
+/install/latest    https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.14.0/strimzi-cluster-operator-0.14.0.yaml     200
```
[the remark this answers is not in the document]
Comment (scholzj) [reply _redirects]: I didn't found one. There is a link to the latest release but not to the latest artifacts.

[the remark this answers is not in the document]
Comment (ppatierno) [reply _redirects]: what do you think about this one?
https://github.com/strimzi/strimzi-kafka-operator/releases/latest/download/strimzi-cluster-operator-0.14.0.yaml
it seems to work

[the remark this answers is not in the document]
Comment (ppatierno) [reply _redirects]: ufff there is 0.14.0 in the YAML name anyway

[the remark this answers is not in the document]
Comment (ppatierno) [reply _redirects]: at this point any solution doesn't make sense, if 0.14.0 isn't in the path, it's in the YAML file anyway :-(

**Pull request merged**